docs/guide/user_management.md

```diff
@@ -17,12 +17,33 @@ You can customize it according to your needs.
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
-### Setting password expiration
+## Password settings
 
-You can set password expiry rules, which will force users to change their passwords periodically.
+You can customize a password policy in your project.
+Each password setting is customizable per User Field Type.
+You can change the [password attributes](#password-attributes) or [password expiration settings](#password-expiration), and determine the rules for [repeating passwords](#repeating-passwords).
 
-Password expiry is defined per User Field Type.
-You can set it for the User Content Type in the `ezuser` Field Type's settings:
+To access the settings, perform the following actions:
+
+1. In the Back Office, in the **Admin** Panel, open the **Content Types** tab.
+1. In the **Content Type Groups** table, click on **Users**.
+1. Edit the User Content Type.
```

Comment:
There can be other CTs that function as users, beyond the built-in User CT (see https://doc.ezplatform.com/en/master/guide/config_repository/#user-identifiers)

Reply:
Can I paste your comment to the doc? Sounds like ready to be included in this section. :)